src/verifysignature.c

```diff
@@ -0,0 +1,113 @@
+// Binary to help us debug the signature verification.
```

Comment:
Can you explain that this is the signature for the not-yet-launched update cache API? We don't want to confuse people who might otherwise expect this to verify SXGs.

Reply:
Done.